pipelines/docker-build-rhtap/docker-build-rhtap.yaml

```diff
@@ -152,66 +152,6 @@ spec:
     workspaces:
     - name: source
       workspace: workspace
-  - name: sast-unicode-check
```

Comment:
these changes already merged as part of [this PR](https://github.com/konflux-ci/build-definitions/pull/1900/files) .

Reply:
Thanks, I rebased it